crates/anvil/src/cmd.rs

```diff
@@ -45,9 +49,23 @@ pub struct NodeArgs {
     pub timestamp: Option<u64>,
 
     /// BIP39 mnemonic phrase used for generating accounts.
-    #[clap(long, short)]
+    /// Cannot be used if `mnemonic_random` or `mnemonic_seed` are used
+    #[clap(long, short, conflicts_with_all = &["mnemonic_seed", "mnemonic_random"])]
     pub mnemonic: Option<String>,
 
+    /// Automatically generates a BIP39 mnemonic phrase, and derives accounts from it.
+    /// Cannot be used with other `mnemonic` options
+    /// You can specify the number of words you want in the mnemonic.
+    /// [default: 12]
+    #[clap(long, conflicts_with_all = &["mnemonic", "mnemonic_seed"], default_missing_value = "12", num_args(0..=1))]
+    pub mnemonic_random: Option<usize>,
+
+    /// Generates a BIP39 mnemonic phrase from a given seed
+    /// CAREFUL: this is obviously NOT SAFE and should only be used for testing
```

Comment:
Hmm, I'm not sure it's a great idea to have this even if we have an unsafe warning, as users won't probably be reading this. how about we just keep `mnemonic_random`?

Reply:
I understand, but I really think it can be a useful feature (it is for me at least). And that's why I only added it to Anvil, and not Cast. Maybe I could print a warning message to the console when this option is used.

Reply:
and I could rename the command `--mnemonic-random-unsafe ` I dont think it could be more clear 😅

Reply:
to me it's no different from the accounts automatically generated by Anvil. It's obvious they shouldnt be used in prod

Reply:
Gotcha, sounds good then. Don't think it hurts—we can reassess later

Reply:
Just curious what is the use case for this feature? Is the idea that anvil's default accounts can have non-null state on live chains which is annoying when forking, so this lets us deterministically get empty accounts? If so that makes sense to me, but wondering if there's other use cases

Reply:
Though I'm not sure what the use case in cast would be